news/2017-07-03-scorewatch-june-week-4.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Scorewatch: June Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+date: 2017-07-03 02:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+---
+ 
+Many impressive first-ever-done scores, first places and more in this week’s article! 
```

Comment:
Hi I'm ZePooNzz and I haz just pushedzz all the fixezzzz but GitHubz wont mark this az rezzzzolved